Dev multi tenant

wyyalt opened the pull request.

tencentblueking-adm commented:
[![CLA assistant check](https://bk-cla.bktencent.com:443/pull/badge/not_signed)](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-sops?pullRequest=7752) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-sops?pullRequest=7752) before we can accept your contribution.<br/>**4** out of **5** committers have signed the CLA.<br/><br/>:white_check_mark: guohelu<br/>:white_check_mark: dengyh<br/>:white_check_mark: wyyalt<br/>:white_check_mark: Mianhuatang8<br/>:x: dannydeng<br/><hr/>**dannydeng** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/bk-sops?pullRequest=7752) it.</sub>

dengyh committed:
Merge pull request #8134 from dengyh/feat/mcp_optimize

feat: 增加判断是否来源于MCP的逻辑 --story=129445652

Mianhuatang8 committed:
fix: TagSelect组件使用远程数据源时，请求支持携带cookie凭证信息 --story=129798291
# Reviewed, transaction id: 70745

Mianhuatang8 committed:
feat: 添加ai小鲸动态上下文 --story=129317277
# Reviewed, transaction id: 70836

Commit:
feat: apigw包下添加get_task_node_log和get_task_plugin_log两个接口--story=130004643

luofann committed:
update: 更新依赖包版本 #ignore

Mianhuatang8 committed:
fix: 正则校验问题修复 --story=129973255
# Reviewed, transaction id: 71286

dengyh committed:
Merge pull request #8144 from kemafd1sf/add_node_log_api_for_apigw

feat: apigw包下添加get_task_node_log和get_task_plugin_log两个接口--story=130004643

luofann committed:
refactor: 去掉移动端package.json中npm的版本声明

luofann committed:
refactor: 更新多租户分支移动端依赖版本过低的问题

Commit:
fix: 修复apigw下两个接口get_task_node_log和get_task_plugin_log两个接口鉴权问题--story=130004643

dengyh committed:
Merge pull request #8147 from kemafd1sf/fix_node_log_api_for_apigw

fix: 修复apigw下两个接口get_task_node_log和get_task_plugin_log两个接口鉴权问题--story=130004643

dengyh committed:
refactor: 清理weixin目录 --story=130071508

luofann committed:
refactor: 升级node-forge版本 #ignore

Mianhuatang8 committed:
feat: 代码框表单增加AI工具 --story=130106980 (#8148)

dengyh committed:
refactor: 更新版本号 --story=130071508

dengyh committed:
Merge pull request #8150 from dengyh/refactor/remove_weixin

refactor: 清理weixin目录 --story=130071508

dengyh committed:
refactor: 优化api-resources.yml中网关API的声明 --story=130158303

dengyh committed:
Merge pull request #8152 from dengyh/refactor/api-resources-update

refactor: 优化api-resources.yml中网关API的声明 --story=130158303

RGardenia committed:
fix: 代码框表单增加AI工具 --story=130162239

dengyh committed:
Merge branch 'master' into release_humming_bird_0918_merge

Mianhuatang8 committed:
fix: 代码框脚本检查自动触发快捷指令 --story=130162239
# Reviewed, transaction id: 72225

Mianhuatang8 committed:
feat: 支持AI排查流程执行失败问题 --story=130202440
# Reviewed, transaction id: 72281

Mianhuatang8 committed:
feat: 支持AI排查流程执行失败问题 --story=130202440
# Reviewed, transaction id: 72282

Mianhuatang8 committed:
fix: 失效样式问题修复 --story=130202440
# Reviewed, transaction id: 72359

luofann committed:
Merge remote-tracking branch 'upstream/release_humming_bird_0918_merge' into merge_master_to_humming_bird

wyyalt committed:
feat: 新增监控api自动授权 #8160

luofann committed:
Merge master into release_humming_bird

luofann committed:
Merge master into release_humming_bird
# Reviewed, transaction id: 72402

luofann committed:
Merge master into release_humming_bird
# Reviewed, transaction id: 72402
# Reviewed, transaction id: 72414

dengyh committed:
refactor: master分支功能同步到hb分支 --story=130255505

dengyh committed:
Merge pull request #8162 from TencentBlueKing/merge_master_to_humming_bird

Merge master to humming bird

luofann committed:
Merge master into release_humming_bird

luofann committed:
Merge master into release_humming_bird
# Reviewed, transaction id: 72472

luofann committed:
Merge master into release_humming_bird
# Reviewed, transaction id: 72472
# Reviewed, transaction id: 72473

luofann committed:
Merge branch 'merge_master_to_humming_bird' of github.com:TencentBlueKing/bk-sops into merge_master_to_humming_bird

dengyh committed:
Merge pull request #8163 from TencentBlueKing/merge_master_to_humming_bird

Merge master to humming bird

dengyh committed:
Merge pull request #8061 from TencentBlueKing/master

定期同步master分支到hb

dengyh committed:
refactor: 处理合并冲突 --story=130255505

dengyh committed:
refactor: 处理从master合入的代码, 使其符合python3.11的规范 --story=130255505

Mianhuatang8 committed:
feat: 多租户计划任务启动时间限制随时区变化 --story=130283561 (#8164)

guohelu committed:
perf: 监控告警屏蔽相关插件优化 --story=129745657
# Reviewed, transaction id: 72639

luofann committed:
fix: 修复encryptRSA导入语句报错的问题

Commit:
feat: 添加获取任务有效时长的的内部接口 --story=130531087

dengyh committed:
Merge pull request #8171 from kemafd1sf/feat/add_get_effective_time_inner

feat: 添加获取任务有效时长的的内部接口 --story=130531087

guohelu committed:
fix: 修复webhook请求参数json格式转换问题 --story=154050993
# Reviewed, transaction id: 73136

dengyh committed:
Merge pull request #8172 from guohelu/master_0130

fix: 修复webhook请求参数json格式转换问题 --story=154050993

dengyh committed:
feat: 构建节点执行的Span --story=130572156

dengyh committed:
refactor: 修复部署上的问题 --story=130255505

dengyh committed:
refactor: 修复部署上的问题 --story=130255505

dengyh committed:
refactor: 修复import的问题 --story=130255505

dengyh committed:
Merge pull request #8166 from guohelu/master_0126

perf: 监控告警屏蔽相关插件优化 --story=129745657

Mianhuatang8 committed:
feat: 多组户增加时区参数 --story=130607311 (#8173)

* feat: 多组户增加时区参数 --story=130607311
# Reviewed, transaction id: 73166

* feat: 多组户增加时区参数 --story=130607311
# Reviewed, transaction id: 73172

* feat: 多组户增加时区参数 --story=130607311
# Reviewed, transaction id: 73176

Mianhuatang8 committed:
feat: 支持AI进行任务总结 --story=130722324
# Reviewed, transaction id: 73326

dengyh committed:
refactor: fix conflict

dengyh committed:
Merge pull request #8174 from dengyh/feat/plugin_trace

feat: 构建节点执行的Span --story=130572156

dengyh committed:
refactor: 修复python版本兼容的问题 --story=130572156

dengyh committed:
Merge pull request #8177 from dengyh/feat/plugin_trace

refactor: 修复python版本兼容的问题 --story=130572156

dengyh committed:
refactor: plugin execute和schedule 的span名称格式修改 --story=130572156

dengyh committed:
Merge pull request #8178 from dengyh/feat/plugin_trace

refactor: plugin execute和schedule 的span名称格式修改 --story=130572156

luofann committed:
Merge branch

dengyh committed:
refactor: 修复github actions中的安全问题 --story=130607311

dengyh committed:
Merge pull request #8179 from TencentBlueKing/dev_multi_tenant_0130_merge_hb

Dev multi tenant 0130 merge hb

Mianhuatang8 committed:
fix: 屏蔽告警插件策略搜索时无法多选问题修复 --story=130827380 (#8181)

* fix: 屏蔽告警插件策略搜索时无法多选问题修复 --story=130827380
# Reviewed, transaction id: 73467

* fix: 优化逻辑避免竞态条件 --story=130827380
# Reviewed, transaction id: 73469

dengyh committed:
Merge branch 'release_humming_bird' into dev_multi_tenant_0203_merge_hb

dengyh committed:
refactor: 处理合并代码的冲突 --story=130607311

dengyh committed:
refactor: 处理合并代码的冲突 --story=130607311

dengyh committed:
Merge pull request #8183 from dengyh/dev_multi_tenant_0203_merge_hb

Dev multi tenant 0203 merge hb

Mianhuatang8 committed:
fix: ai小鲸回填数据获取时机问题修复 --story=130202440 (#8182)

* fix: ai小鲸回填数据获取时机问题修复 --story=130202440
# Reviewed, transaction id: 73494

* fix: 模板和任务code无需脚本检查 --story=130202440

* fix: 模板和任务code无需脚本检查 --story=130202440
# Reviewed, transaction id: 73503

* fix: 删除重复初始化 --story=130202440

dengyh committed:
refactor: 修复内置插件span名称不对的问题 & span层级问题修复 --story=130572156

Mianhuatang8 committed:
fix: 多组户增加时区参数 --story=130607311
# Reviewed, transaction id: 73647

dengyh committed:
Merge pull request #8184 from dengyh/feat/plugin_trace_v2

refactor: 修复内置插件span名称不对的问题 & span层级问题修复 --story=130572156

guohelu committed:
fix: 修复迁移文件冲突 --story=130069955
# Reviewed, transaction id: 73673

wyyalt committed:
Merge pull request #8187 from guohelu/dev_multi_tenant_0204

fix: 修复迁移文件冲突 --story=130069955

Mianhuatang8 committed:
fix: 正则校验配置导致节点无法保存 --story=131004484
# Reviewed, transaction id: 73712

Mianhuatang8 committed:
fix: 节点输出参数勾选的全局变量支持手动删除  --story=130415920 (#8188)

* fix: 节点输出参数勾选的全局变量支持手动删除  --story=130415920
# Reviewed, transaction id: 73667

* fix: 节点输出参数勾选的全局变量支持手动删除  --story=130415920
# Reviewed, transaction id: 73707

wyyalt committed:
fix:周期任务&计划任务返回增加timezone参数 --story=130607311

Mianhuatang8 committed:
fix: 校验规则补充字符串判断 --story=131004484
# Reviewed, transaction id: 73743

Mianhuatang8 committed:
fix: 多组户增加时区参数 --story=130607311

Mianhuatang8 committed:
fix: 逻辑优化 --story=130607311
# Reviewed, transaction id: 73791

Mianhuatang8 committed:
fix: 多组户增加时区参数 --story=130607311 (#8195)

* fix: 多组户增加时区参数 --story=130607311
# Reviewed, transaction id: 73790

* fix: 逻辑优化 --story=130607311
# Reviewed, transaction id: 73791

wyyalt committed:
fix:修复周期任务更新时区不对问题 --story=130607311

dengyh committed:
refactor: 修复部分插件没有继承BasePluginService的问题 & 修复部分插件execute或者schedule返回None导致认为插件执行失败的问题 --story=130572156

dengyh committed:
refactor: 修复execute和schedule的父span不对的问题 --story=130572156

dengyh committed:
Merge pull request #8196 from dengyh/feat/plugin_trace_v3

Feat/plugin trace v3

dengyh committed:
refactor: approve插件补充调用finish_schedule --story=130572156

dengyh committed:
Merge pull request #8197 from dengyh/feat/plugin_trace_v3

refactor: approve插件补充调用finish_schedule --story=130572156

dengyh committed:
refactor: 调整span属性的内容 --story=130572156

dengyh committed:
Merge pull request #8198 from dengyh/feat/plugin_trace_v3

refactor: 调整span属性的内容 --story=130572156

dengyh committed:
refactor: 流程结束后, 流程output清理span相关的内置属性 --story=130572156

dengyh committed:
Merge pull request #8203 from dengyh/feat/plugin_trace_v3

refactor: 流程结束后, 流程output清理span相关的内置属性 --story=130572156

Commit:
feat: 增加AI分析报告企业微信推送功能 --story=131238327

Commit:
feat: 添加获取JOB节点的执行日志的内部接口 --story=131632907

guohelu committed:
feat: 修改数据库host定义 --story=130307462
# Reviewed, transaction id: 74440

guohelu committed:
fix: 修改review问题 --story=130307462
# Reviewed, transaction id: 74445

guohelu committed:
fix: 修改host变量名 --story=130307462
# Reviewed, transaction id: 74468

wyyalt committed:
Merge pull request #8208 from guohelu/dev_multi_tenant_0227

feat: 修改数据库host定义 --story=130307462

dengyh committed:
Merge pull request #8202 from kemafd1sf/feat/add_ai_analysis_notify

Feat/add ai analysis notify

Mianhuatang8 committed:
fix: 节点详情预览图中状态显示错误问题修复 --story=131742471 (#8210)

* fix: 节点详情预览图中状态显示错误问题修复 --story=131742471
# Reviewed, transaction id: 74483

* fix: 节点详情预览图中状态显示错误问题修复 --story=131742471
# Reviewed, transaction id: 74489

Mianhuatang8 committed:
feat: 流程基础信息增加AI分析 --story=131224849 (#8206)

* feat: 流程基础信息增加AI分析 --story=131224849

* feat: 流程基础信息增加AI分析 --story=131224849
# Reviewed, transaction id: 74201

* fix: 接口字段修改 --story=131224849
# Reviewed, transaction id: 74544

* fix: 兼容性处理 --story=131224849
# Reviewed, transaction id: 74549

* fix: 兼容性处理 --story=131224849
# Reviewed, transaction id: 74550

Commit:
fix: 修改AI通知开关 --story=131238327

dengyh committed:
Merge pull request #8211 from kemafd1sf/feat/add_ai_analysis_notify

fix: 修改AI通知开关 --story=131238327

Commit:
fix: 修改AI通知任务task_id获取逻辑 --story=131238327

dengyh committed:
Merge pull request #8212 from kemafd1sf/feat/add_ai_analysis_notify

fix: 修改AI通知任务task_id获取逻辑 --story=131238327

dengyh committed:
Merge pull request #8209 from kemafd1sf/feat/add_inner_api_for_get_job_log

feat: 添加获取JOB节点的执行日志的内部接口 --story=131632907

Commit:
refactor: ai通知逻辑完善日志 --story=131238327

dengyh committed:
Merge pull request #8213 from kemafd1sf/feat/add_ai_analysis_notify

refactor: ai通知逻辑完善日志 --story=131238327

Commit:
fix: 添加AI通知独立worker配置 --story=131238327

dengyh committed:
Merge pull request #8214 from kemafd1sf/feat/add_ai_analysis_notify

fix: 添加AI通知独立worker配置 --story=131238327

kemafd1sf committed:
fix: 添加ai错误分析报告内容过长截断逻辑 --story=131238327

kemafd1sf committed:
fix: 企业微信个人通知修改为仅支持通知执行人 --story=131238327

Mianhuatang8 committed:
feat: AI分析设计界面重构 --story=131224849 (#8216)

* feat: AI分析设计界面重构 --story=131224849
# Reviewed, transaction id: 74883

* fix: 逻辑优化 --story=131224849
# Reviewed, transaction id: 74888

* fix: 减少重复请求 --story=131224849
# Reviewed, transaction id: 74897

* fix: 逻辑优化 --story=131224849
# Reviewed, transaction id: 74898

dengyh committed:
docs: add design doc for get_template_info unfold_subprocess parameter

新增设计文档，说明在 get_template_info 接口中增加 unfold_subprocess
参数的方案设计，用于一次性返回包含所有子流程完整配置的流程树结构。

Made-with: Cursor

dengyh committed:
docs: add implementation plan for get_template_info unfold_subprocess

详细实现计划，包含 5 个任务：serializer 新增字段、replace_template_id_recursive
工具函数、format_template_data 分支、视图层异常处理、文档更新。

Made-with: Cursor

dengyh committed:
feat: add unfold_subprocess field to IncludeTemplateSerializer

Made-with: Cursor

dengyh committed:
feat: add replace_template_id_recursive utility function

Adds recursive ID conversion for pipeline trees that contain unfolded
subprocess nodes (act["pipeline"]), where the existing replace_template_id
only processes the top-level activities.

Made-with: Cursor

dengyh committed:
Merge pull request #8215 from kemafd1sf/feat/add_ai_analysis_notify

fix: 添加ai错误分析报告内容过长截断逻辑和企业微信个人通知目前仅支持任务执行人 --story=131238327

dengyh committed:
fix: use COMMON constant instead of magic string in replace_template_id_recursive

Made-with: Cursor

dengyh committed:
feat: support unfold_subprocess in format_template_data

Made-with: Cursor

dengyh committed:
fix: use safe pop(key, None) in both format_template_data branches

Made-with: Cursor

dengyh committed:
feat: get_template_info supports unfold_subprocess parameter

Reads unfold_subprocess from serializer validated data, passes it to
format_template_data, and wraps the call in a try/except so that any
exception (e.g. recursion limit from PipelineException) returns a
structured result=False response instead of a 500.

Made-with: Cursor

dengyh committed:
fix: use accurate error message in get_template_info exception handler

Replace "unfold_subprocess" with "format_template_data" in the except block
log and response message, since the handler catches any exception from
format_template_data, not just unfold_subprocess errors. Update the
corresponding test assertion and add a code field assertion.

Made-with: Cursor

dengyh committed:
docs: update get_template_info docs and skill with unfold_subprocess parameter

Made-with: Cursor

dengyh committed:
Revert "docs: update get_template_info docs and skill with unfold_subprocess parameter"

This reverts commit a432e71e19e6c6d560791c201bae4d17ef76970f.

dengyh committed:
Merge pull request #8217 from dengyh/feat/get-template-info-unfold-subprocess

feat: get_template_info 接口新增 unfold_subprocess 参数，支持递归展开子流程配置

dengyh committed:
docs: add unfold_subprocess parameter to get_template_info API docs

Made-with: Cursor

kemafd1sf committed:
fix: ai报告通知截断兜底逻辑长度限制由字符修改为字节 --story=131238327

dengyh committed:
Merge pull request #8219 from kemafd1sf/feat/add_ai_analysis_notify

fix: ai报告通知截断兜底逻辑长度限制由字符修改为字节 --story=131238327

dengyh committed:
Merge pull request #8218 from dengyh/feat/get-template-info-unfold-subprocess

docs: add unfold_subprocess parameter to get_template_info API docs

Mianhuatang8 committed:
fix: 任务详情头部样式问题修复 --story=132219547 (#8220)

* fix: 任务详情头部样式问题修复 --story=132219547
# Reviewed, transaction id: 75236

* fix: 删除冗余样式 --story=132219547
# Reviewed, transaction id: 75237

dengyh committed:
fix: 周期任务开关操作时刷新更新人 --bug=132378020

Made-with: Cursor

RGardenia committed:
feat: 智能体流程编排 --story=131050991

dengyh committed:
feat: Job 插件支持节点名+时间戳作为 job 任务名 --story=132389598

- 新增 get_job_task_name 工具函数
- 快速执行脚本 v2.1、快速分发文件 v3.1、执行作业 v2.1
- 跨业务快速执行脚本 v1.2、跨业务快速分发文件 v1.2、跨业务执行作业 v1.2

Made-with: Cursor

dengyh committed:
refactor: 优化测试用例 --story=132331359

dengyh committed:
Merge pull request #8227 from dengyh/refactor/unittest_optimize

refactor: 优化测试用例 --story=132331359

kemafd1sf committed:
feat: 支持多种JOB节点查询日志 --story=131632907

dengyh committed:
Merge pull request #8223 from dengyh/fix/periodic-task-editor-update

fix: 周期任务开关操作时刷新更新人 --bug=132378020

dengyh committed:
refactor: 移除 all_biz 插件的节点任务名支持，仅保留 3 个核心插件 --story=132389598

Made-with: Cursor

dengyh committed:
Merge pull request #8226 from dengyh/feature/job-task-name-node

支持使用节点名称来作为job插件节点的任务名称

RGardenia committed:
fix: 流程编排鉴权 --story=131050991

dengyh committed:
Merge pull request #8228 from kemafd1sf/feat/add_all_job_plugin_node_get_job_log

feat: 支持多种JOB节点查询日志 --story=131632907

dengyh committed:
Merge pull request #8225 from RGardenia/master

feat: 智能体流程编排 --story=131050991

dengyh committed:
refactor: 优化部分内部apigw url结构 --story=132544238

dengyh committed:
Merge pull request #8229 from dengyh/refactor/optimize_apigw_configs

refactor: 优化部分内部apigw url结构 --story=132544238

dengyh committed:
fix: 修复pipeline plugin import 的问题 --bug=132544238

dengyh committed:
Merge pull request #8230 from dengyh/fix/pipeline_plugins_import

fix: 修复pipeline plugin import 的问题 --bug=132544238

dengyh committed:
refactor: 完善node_callback 文档 --story=132544238

wyyalt committed:
Merge pull request #8232 from TencentBlueKing/noah_multi_tenant

Noah multi tenant

dengyh committed:
Merge pull request #8231 from dengyh/refactor/optimize_callback_docs

refactor: 完善node_callback 文档 --story=132544238

dengyh committed:
refactor: 修复job新插件版本导致的import问题 --story=132389598

dengyh committed:
Merge pull request #8234 from dengyh/fix/job_plugin_import

refactor: 修复job新插件版本导致的import问题 --story=132389598

Mianhuatang8 committed:
fix: ai小鲸icon功能支持开关  --story=132588811 (#8233)

* fix: ai小鲸icon功能支持开关  --story=132588811
# Reviewed, transaction id: 75668

* fix: 开关状态统一管理 --story=132588811
# Reviewed, transaction id: 75670

* fix: 开关状态统一管理 --story=132588811
# Reviewed, transaction id: 75698

dengyh committed:
refactor: 网关接口作为mcp服务时, 返回pipelien_tree进行裁剪 --story=132636521

dengyh committed:
Merge pull request #8235 from dengyh/feat/pipeline-tree-trim

refactor: 网关接口作为mcp服务时, 返回pipelien_tree进行裁剪 --story=132636521

RGardenia committed:
fix: 流程编排 元数据默认值设置 --story=131050991 (#8236)

* fix: 流程编排 元数据默认值设置 --story=131050991

* fix: 流程编排 元数据默认值 mutations 设置 --story=131050991

* fix: 流程编排 元数据重置 --story=131050991

guohelu committed:
fix: 修复单租户测试缺陷 --story=132553972
# Reviewed, transaction id: 75844

RGardenia committed:
feat: 创建模版网关接口 --story=131050991

Mianhuatang8 committed:
feat: 流程编辑页面增加“历史任务”的快捷入口 --story=132713061 (#8237)

* feat: 流程编辑页面增加“历史任务”的快捷入口 --story=132713061
# Reviewed, transaction id: 75838

* feat: 流程编辑页面增加“历史任务”的快捷入口 --story=132713061
# Reviewed, transaction id: 75845

wyyalt committed:
Merge pull request #8238 from guohelu/dev_multi_tenant_0316

fix: 修复单租户测试缺陷 --story=132553972

RGardenia committed:
fix: 创建模版网关接口 排版 --story=131050991

guohelu committed:
fix: 修改版本号 --story=132553972
# Reviewed, transaction id: 75889

wyyalt committed:
Merge pull request #8239 from guohelu/dev_multi_tenant_260316

fix: 修改版本号 --story=132553972

kemafd1sf committed:
fix: 修复网关API返回DRF Response导致500 错误 --story=132739429

dengyh committed:
Merge pull request #8240 from kemafd1sf/fix/api_response_error

fix: 修复网关API返回DRF Response导致500 错误 --story=132739429

RGardenia committed:
docs: 创建模版网关接口文档 --story=131050991

dengyh committed:
Merge pull request #8241 from RGardenia/gardenia/create_template_api

feat: 创建模版网关接口

dengyh committed:
feat: get_template_info 网关接口支持 format=yaml 参数导出 YAML 格式 pipeline_tree --story=132767026

dengyh committed:
feat: create_template 网关接口支持 format=yaml 参数导入 YAML 格式流程 --story=132767026

Made-with: Cursor

dengyh committed:
refactor: 补充相关开发设计 --story=132767026

dengyh committed:
fix: 修复 PR review 问题：template_order 越界保护、多模板警告日志、文档补充说明 --story=132767026

Made-with: Cursor

dengyh committed:
Merge pull request #8243 from dengyh/feat/get-template-info-yaml-format

feat: get_template_info 和 create_template 网关接口支持 format=yaml 参数导出 YAML 格式 pipeline_tree…

dengyh committed:
fix: yaml文件默认不提交 --bug=132767026

guohelu committed:
feat: 增加数据库环境配置 --story=130307462
# Reviewed, transaction id: 76080

wyyalt committed:
Merge pull request #8245 from guohelu/dev_multi_tenant_0318

feat: 增加数据库环境配置 --story=130307462

dengyh committed:
Merge remote-tracking branch 'upstream/master' into release_humming_bird_0319_merge

Made-with: Cursor

# Conflicts:
#	.github/workflows/unittest.yml
#	app_desc.yaml
#	frontend/desktop/src/pages/task/TaskExecute/ExecuteInfo.vue
#	frontend/desktop/src/pages/task/TaskExecute/ExecuteInfo/components/NodeExecuteInfo/components/ExecuteRecord/index.vue
#	frontend/desktop/src/pages/task/TaskExecute/TaskOperationHeader.vue
#	frontend/desktop/src/pages/template/TemplateEdit/TemplateSetting/TabTemplateConfig.vue
#	frontend/desktop/src/store/index.js
#	gcloud/apigw/docs/apigw-docs.tgz
#	gcloud/apigw/urls.py
#	gcloud/apigw/views/utils.py
#	gcloud/taskflow3/celery/tasks.py
#	gcloud/taskflow3/domains/dispatchers/task.py
#	gcloud/tasktmpl3/apis/django/api.py
#	gcloud/tasktmpl3/urls.py
#	pipeline_plugins/components/collections/common.py
#	pipeline_plugins/components/collections/controller.py
#	pipeline_plugins/components/collections/http/v1_0.py
#	pipeline_plugins/components/collections/sites/open/bk/approve/v1_0.py
#	pipeline_plugins/components/collections/sites/open/bk/display/v1_0.py
#	pipeline_plugins/components/collections/sites/open/bk/notify/legacy.py
#	pipeline_plugins/components/collections/sites/open/bk/notify/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/base.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_delete_set/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_delete_set/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_module_update/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_transfer_host_module/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_transfer_host_module/v1_1.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_update_set/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/create_module/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/create_set/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/create_set/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/create_set/v2_0.py
#	pipeline_plugins/components/collections/sites/open/cc/create_set_by_template/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/empty_set_hosts/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/empty_set_hosts/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/host_custom_property_change/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/host_lock/base.py
#	pipeline_plugins/components/collections/sites/open/cc/replace_fault_machine/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/transfer_to_recovery/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/update_module/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/update_module/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/update_set_service_status/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/update_set_service_status/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/update_set_service_status/v1_1.py
#	pipeline_plugins/components/collections/sites/open/job/cron_task/legacy.py
#	pipeline_plugins/components/collections/sites/open/job/execute_task/v1_2.py
#	pipeline_plugins/components/collections/sites/open/job/fast_execute_script/legacy.py
#	pipeline_plugins/components/collections/sites/open/job/fast_push_file/legacy.py
#	pipeline_plugins/components/collections/sites/open/job/fast_push_file/v2_1.py
#	pipeline_plugins/components/collections/sites/open/job/fetch_task_log/v1_0.py
#	pipeline_plugins/components/collections/sites/open/job/local_content_upload/base_service.py
#	pipeline_plugins/components/collections/sites/open/job/push_local_files/base_service.py
#	pipeline_plugins/components/collections/sites/open/job/push_local_files/v1_0_0.py
#	pipeline_plugins/components/collections/sites/open/monitor/alarm_shield/v1_0.py
#	pipeline_plugins/components/collections/sites/open/monitor/alarm_shield_disable/v1_0.py
#	pipeline_plugins/components/collections/sites/open/monitor/alarm_shield_strategy/v1_0.py
#	pipeline_plugins/components/collections/sites/open/monitor/alarm_shield_strategy/v1_1.py
#	pipeline_plugins/components/collections/sites/open/monitor/alarm_shield_strategy/v1_2.py
#	pipeline_plugins/components/collections/sites/open/monitor/alarm_shield_strategy/v2_0.py
#	pipeline_plugins/components/collections/sites/open/monitor/base.py
#	pipeline_plugins/components/collections/sites/open/nodeman/base.py
#	pipeline_plugins/components/query/sites/open/monitor.py
#	pipeline_plugins/components/utils/sites/open/utils.py

dengyh committed:
Merge branch 'release_humming_bird_0319_merge' into dev_multi_tenant_0319_merge

Made-with: Cursor

# Conflicts:
#	app_desc.yaml
#	docs/apidoc/zh/create_template.md
#	docs/apidoc/zh/get_node_job_executed_log.md
#	docs/apidoc/zh/get_plugin_base_info.md
#	docs/apidoc/zh/get_task_node_log.md
#	docs/apidoc/zh/get_task_plugin_log.md
#	frontend/desktop/src/store/index.js
#	gcloud/apigw/docs/apigw-docs.tgz
#	gcloud/apigw/views/fast_create_task.py
#	gcloud/iam_auth/view_interceptors/template.py
#	gcloud/taskflow3/signals/handlers.py
#	pipeline_plugins/components/collections/controller.py
#	pipeline_plugins/components/collections/gse_kit/flush_process/v1_0.py
#	pipeline_plugins/components/collections/gse_kit/job_exec/v1_0.py
#	pipeline_plugins/components/collections/sites/open/bk/approve/v1_0.py
#	pipeline_plugins/components/collections/sites/open/bk/notify/legacy.py
#	pipeline_plugins/components/collections/sites/open/bk/notify/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/base.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_delete_set/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_delete_set/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_module_update/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_transfer_host_module/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_transfer_host_module/v1_1.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_update_host/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/batch_update_set/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/create_module/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/create_set/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/create_set/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/create_set/v2_0.py
#	pipeline_plugins/components/collections/sites/open/cc/create_set_by_template/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/empty_set_hosts/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/empty_set_hosts/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/host_custom_property_change/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/host_lock/base.py
#	pipeline_plugins/components/collections/sites/open/cc/replace_fault_machine/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/transfer_host_module/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/transfer_host_module/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/update_host/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/update_module/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/update_module/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/update_set/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/update_set/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/update_set_service_status/legacy.py
#	pipeline_plugins/components/collections/sites/open/cc/update_set_service_status/v1_0.py
#	pipeline_plugins/components/collections/sites/open/cc/update_set_service_status/v1_1.py
#	pipeline_plugins/components/collections/sites/open/job/all_biz_fast_push_file/base_service.py
#	pipeline_plugins/components/collections/sites/open/job/base.py
#	pipeline_plugins/components/collections/sites/open/job/cron_task/legacy.py
#	pipeline_plugins/components/collections/sites/open/job/execute_task/execute_task_base.py
#	pipeline_plugins/components/collections/sites/open/job/fast_push_file/v2_1.py
#	pipeline_plugins/components/collections/sites/open/job/fetch_task_log/v1_0.py
#	pipeline_plugins/components/collections/sites/open/job/local_content_upload/base_service.py
#	pipeline_plugins/components/collections/sites/open/monitor/alarm_shield_disable/v1_0.py
#	pipeline_plugins/components/collections/sites/open/monitor/base.py
#	pipeline_plugins/components/collections/sites/open/nodeman/base.py
#	pipeline_plugins/components/collections/sites/open/wechat_work/wechat_work_send_message/v1_0.py
#	pipeline_plugins/components/query/sites/open/monitor.py
#	pipeline_plugins/components/utils/sites/open/utils.py

RGardenia committed:
feat: 流程编排添加功能开关逻辑 (#8244)

* feat: 流程编排添加功能开关逻辑 --story=131050991

* fix: 错误码修改 --story=131050991

* feat: 流程编排添加功能开关逻辑 前端 --story=131050991

dengyh committed:
Merge remote-tracking branch 'upstream/master' into release_humming_bird_0319_merge

dengyh committed:
Merge branch 'release_humming_bird_0319_merge' into dev_multi_tenant_0319_merge

dengyh committed:
fix: 修复 AI 通知任务的 celery 装饰器兼容性（@task → @current_app.task）

Made-with: Cursor

dengyh committed:
Merge branch 'release_humming_bird_0319_merge' into dev_multi_tenant_0319_merge

dengyh committed:
fix: 修复 monitor.py 中 django.conf.urls.url 的 Django 4.x 兼容性问题（url → re_path）

Made-with: Cursor

dengyh committed:
Merge branch 'release_humming_bird_0319_merge' into dev_multi_tenant_0319_merge

Made-with: Cursor

# Conflicts:
#	pipeline_plugins/components/query/sites/open/monitor.py

dengyh committed:
fix: 修复审批插件合并后的测试用例兼容性问题

Made-with: Cursor

dengyh committed:
Merge pull request #8247 from dengyh/dev_multi_tenant_0319_merge

chore: sync release_humming_bird to dev_multi_tenant 20250319

guohelu committed:
fix: 修复代码合并问题 --story=132553972
# Reviewed, transaction id: 76229

wyyalt committed:
Merge pull request #8251 from guohelu/dev_multi_tenant_0319

fix: 修复代码合并问题 --story=132553972

Mianhuatang8 committed:
fix: master同步多租户代码合并问题 --story=132896328 (#8255)

* fix: master同步多租户代码合并问题 --story=132896328
# Reviewed, transaction id: 76412

* fix: 删除无效代码 --story=132896328
# Reviewed, transaction id: 76418

dengyh committed:
fix: 修复 release_humming_bird 合入 dev_multi_tenant 时的合并遗留问题

1. controller.py: pytz.timezone 改为 timezone.pytz.timezone（import 已移除但用法未同步）
2. base_service.py: plugin_schedule 中 get_client_by_user 改为 get_client_by_username
3. get_node_job_executed_log.py: ESB 客户端改为 API Gateway 客户端，补充 tenant_id 传递
4. fast_execute_script/v2_1.py: 适配多租户（API Gateway 客户端 + tenant_id header）
5. fast_push_file/v3_1.py: 适配多租户（API Gateway 客户端 + tenant_id header）

--story=132553972

Made-with: Cursor

dengyh committed:
Merge pull request #8256 from dengyh/fix/merge-issues-cleanup

fix: 修复 release_humming_bird 合入 dev_multi_tenant 时的合并遗留问题

dengyh committed:
fix: 修复 controller.py Django 4.2 时区兼容性问题

Django 4.2 移除了 django.utils.timezone 中对 pytz 的 import，
timezone.pytz.timezone() 运行时报 AttributeError，改为直接 import pytz

--story=132553972

Made-with: Cursor

dengyh committed:
Merge pull request #8257 from dengyh/fix/merge-issues-cleanup

fix: 修复 controller.py 时区兼容性和任务详情页操作按钮不显示

guohelu committed:
fix: 修复单租户测试缺陷 --story=132553972
# Reviewed, transaction id: 76544

wyyalt committed:
Merge pull request #8261 from guohelu/dev_multi_tenant_0324

fix: 修复单租户测试缺陷 --story=132553972

Mianhuatang8 committed:
fix: 多租户分支同步master最新功能后测试问题修复 --story=132934185 (#8262)

* fix: 职能化-选择流程模板不正确 --bug=155923801 --story=132934185

* fix: 周期任务/计划任务通知方式展示缺失 --bug=155924533 --story=132934185

* fix: 后台管理-编辑周期任务报错 --bug=155923407 --story=132934185
# Reviewed, transaction id: 76587

dengyh committed:
fix: 修复 get_target_server_hybrid_with_host_id 缺少 tenant_id 导致插件执行报错

master 合入多租户分支后，get_target_server_hybrid_with_host_id 方法未适配
tenant_id 参数，导致内部调用 get_target_server_ipv6_across_business 等方法
时参数错位，快速分发文件等插件执行时抛出 TypeError。

Made-with: Cursor

dengyh committed:
Merge pull request #8263 from TencentBlueKing/fix/multi-tenant-hybrid-host-id-tenant-id

fix: 修复 get_target_server_hybrid_with_host_id 缺少 tenant_id 导致插件执行报错

Mianhuatang8 committed:
fix: 轻应用创建任务人名称展示不对 --bug=155923954  --story=132934185
# Reviewed, transaction id: 76608

Mianhuatang8 committed:
fix: 补充名称转换 --bug=155923954  --story=132934185
# Reviewed, transaction id: 76636

guohelu committed:
fix: 修复租户初始化参数问题 --story=132553972
# Reviewed, transaction id: 76637

wyyalt committed:
Merge pull request #8267 from guohelu/dev_multi_tenant_260324

fix: 修复租户初始化参数问题 --story=132553972

guohelu committed:
fix: 修复插件问题 --story=132553972
# Reviewed, transaction id: 76651

wyyalt committed:
Merge pull request #8268 from guohelu/dev_multi_tenant_324

fix: 修复插件问题 --story=132553972

guohelu committed:
fix: 修复租户初始化参数问题 --stor
# Reviewed, transaction id: 76676

wyyalt committed:
Merge pull request #8269 from guohelu/dev_multi_tenant_24

fix: 修复租户初始化参数问题 --stor

guohelu committed:
fix: 修复租户初始化问题 --story=132553972
# Reviewed, transaction id: 76772

wyyalt committed:
Merge pull request #8271 from guohelu/dev_multi_tenant_0325

fix: 修复租户初始化问题 --story=132553972

guohelu committed:
feat: 增加标准运维迁移工具和说明 --story=133074067
# Reviewed, transaction id: 77233

guohelu committed:
fix: 修改租户ID硬编码配置 --story=133074067
# Reviewed, transaction id: 77238

wyyalt committed:
Merge pull request #8276 from guohelu/dev_multi_tenant_0331

feat: 增加标准运维迁移工具和说明 --story=133074067

guohelu committed:
fix: 修复插件分组问题 --story=133074067
# Reviewed, transaction id: 77275

guohelu committed:
feat: 补充网关资源app主动授权 --story=133074067

wyyalt committed:
Merge pull request #8277 from guohelu/dev_multi_tenant_331

fix: 修复插件分组问题 --story=133074067

dengyh committed:
fix: 修复远程插件包源管理更新时 tenant_id 透传到 pipeline 模型导致报错 --bug=156296546

Made-with: Cursor

guohelu committed:
feat: 远程插件请求补充租户信息 --story=133074067
# Reviewed, transaction id: 77364

wyyalt committed:
Merge pull request #8279 from dengyh/fix/external-plugins-tenant-id-leak

fix: 修复远程插件包源管理更新时 tenant_id 透传到 pipeline 模型导致报错

wyyalt committed:
Merge pull request #8280 from guohelu/dev_multi_tenant_033

feat: 远程插件请求补充租户信息 --story=133074067

dengyh committed:
fix: 为远程插件包源添加 Django 4.x 兼容补丁解决旧插件无法加载问题 --bug=156300881

Made-with: Cursor

wyyalt committed:
Merge pull request #8281 from dengyh/fix/external-plugins-django4-compat

fix: 为远程插件包源添加 Django 4.x 兼容补丁解决旧插件无法加载问题

dengyh committed:
feat: 远程插件(蓝鲸标准插件)多租户适配，传递X-Bk-Tenant-Id并合并全租户与单租户插件列表 --story=133182629

多租户模式下插件列表接口分别请求system和用户租户的插件并合并返回，
其他插件接口根据app_tenant_mode动态确定tenant_id，前端展示全租户标签。

Made-with: Cursor

dengyh committed:
fix: 插件列表翻页增加兜底退出条件，app_tenant_mode改为无条件赋值 --story=133182629

Made-with: Cursor

dengyh committed:
Merge pull request #8282 from dengyh/feat/remote-plugin-multi-tenant

feat: 远程插件(蓝鲸标准插件)多租户适配，传递X-Bk-Tenant-Id并合并全租户与单租户插件列表

Mianhuatang8 committed:
fix: 多租户bk2game环境插件用户名转换 --bug=156394455 (#8286)

* fix: 多租户bk2game环境插件用户名转换 --bug=156394455
# Reviewed, transaction id: 77689

* fix: 多租户bk2game环境插件用户名转换 --bug=156394455
# Reviewed, transaction id: 77690

guohelu committed:
fix: 调整数据同步脚本 --story=133074067
# Reviewed, transaction id: 77804

guohelu committed:
fix: 回调请求增加过滤条件 --story=133074067
# Reviewed, transaction id: 77832

dengyh committed:
fix: 远程插件(第三方插件)执行时未传递tenant_id导致跨租户调用被拒绝 --story=133182629

RemotePluginService的plugin_execute和plugin_schedule在构造PluginServiceApiClient时
未从parent_data获取tenant_id，导致APIGW请求header中X-Bk-Tenant-Id回退为默认值default，
与网关所属租户不匹配触发CROSS_TENANT_FORBIDDEN

Made-with: Cursor

dengyh committed:
Merge pull request #8288 from dengyh/fix/remote-plugin-tenant-id

fix: 远程插件执行时未传递tenant_id导致跨租户调用被拒绝

guohelu committed:
fix: 调整租户报错信息返回 --story=133074067
# Reviewed, transaction id: 77954

wyyalt committed:
Merge pull request #8287 from guohelu/dev_multi_tenant_0407

fix: 调整数据同步脚本 --story=133074067

Mianhuatang8 committed:
feat: 多租户右上角用户信息统一组件以及优化报错信息 --story=132714198 --story=132714178 (#8289)

* feat: 多租户右上角用户信息统一组件以及优化报错信息 --story=132714198 --story=132714178
# Reviewed, transaction id: 77964

* fix: 修改空间报错信息偏移量 --story=132714178
# Reviewed, transaction id: 77969

Mianhuatang8 committed:
fix: 样式问题修复 --story=132714178
# Reviewed, transaction id: 77990

Mianhuatang8 committed:
fix: 公共流程配置执行代理人无法保存问题修复 #ignore
# Reviewed, transaction id: 78079

Mianhuatang8 committed:
fix: 用户信息补充图标 --story=132714198 (#8294)

* fix: 用户信息补充图标 --story=132714198
# Reviewed, transaction id: 78130

* fix: 用户信息补充图标 --story=132714198

Add missing newline at the end of the file.

guohelu committed:
fix: 修复应用态接口遗漏问题 --story=133074067

wyyalt committed:
Merge pull request #8297 from guohelu/dev_multi_tenant_0413

fix: 修复应用态接口遗漏问题 --story=133074067

guohelu committed:
fix: 修改变量描述信息 --story=133074067
# Reviewed, transaction id: 78581

wyyalt committed:
Merge pull request #8307 from guohelu/dev_multi_tenant_0416

fix: 修改变量描述信息 --story=133074067

Mianhuatang8 committed:
fix: 多租户日期时间类型删除时区 --story=133562190
# Reviewed, transaction id: 78613

Commit:
fix: cmsi通知支持动态渠道发送 --bug=1010131351157072426

Commit:
fix: 升级protobuf到5.29.6修复安全风险 --bug=1010131351157076013

dengyh committed:
Merge pull request #8311 from dengyh/fix/cmsi-dynamic-msg-types

fix: cmsi通知支持动态渠道发送

guohelu committed:
fix: 修复跨业务导入流程问题 --story=133074067
# Reviewed, transaction id: 81084

wyyalt committed:
Merge pull request #8359 from guohelu/dev_multi_tenant_0528

fix: 修复跨业务导入流程问题 --story=133074067

guohelu committed:
feat: 补充流程覆盖更新脚本 --story=134659099
# Reviewed, transaction id: 81421

guohelu committed:
fix: 修改更新逻辑 --story=134659099
# Reviewed, transaction id: 81426

guohelu committed:
feat: 补充更新校验问题 --story=134659099
# Reviewed, transaction id: 81472

dengyh committed:
Merge pull request #8369 from guohelu/dev_multi_tenant_0603

feat: 补充流程覆盖更新脚本 --story=134659099

guohelu committed:
fix: 补充流程查询条件 --story=134659099
# Reviewed, transaction id: 81483

dengyh committed:
Merge pull request #8372 from guohelu/dev_multi_tenant_0604

fix: 补充流程查询条件 --story=134659099

guohelu committed:
fix: 修改bkchat开关控制逻辑 --story=134659099
# Reviewed, transaction id: 81800

wyyalt committed:
Merge pull request #8378 from guohelu/dev_multi_tenant_0610

fix: 修改bkchat开关控制逻辑 --story=134659099

dengyh commented:
多租户归档建议：该 PR 更像长期集成/跟踪分支，建议作为 multi-tenant 主跟踪入口保留；后续测试修复类 PR 统一汇总到该分支或新的主跟踪 PR。
